src/pages/About/MissionAndPurpose/MissionAndPurpose.js

```diff
@@ -0,0 +1,47 @@
+import React from 'react';
+// import PropTypes from 'prop-types'
+import Container from '../../../components/Container/Container.js'
```

Comment:
```suggestion
import Container from 'components/Container/Container';
```
write the absolute path please.